File: 0283-move-zeroes/0283-move-zeroes.py
class Solution(object):
    def moveZeroes(self, nums):
        n = len(nums);
        count = 0;
        for i in range(n):
            if nums[i] != 0:
                nums[count] = nums[i];
                count+=1;
        while count < n :
            nums[count] = 0;
            count+=1;
        """
        :type nums: List[int]
        :rtype: None Do not return anything, modify nums in-place instead.
        """
        

File: 0283-move-zeroes/0283-move-zeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int co = 0;
        for(int n:nums){
            if(n != 0){
                nums[co] = n;
                co++;
            }
        }
        int p = nums.size();
        while(co<p){
            nums[co] = 0;
            co++;
        }
        

    }
};